backend/app/utils.py

```
@@ -184,6 +184,29 @@ def generate_new_account_email(
     return EmailData(html_content=html_content, subject=subject)
 
 
+def generate_invite_email(
+    *,
+    email_to: str,
```

Ruff (0.15.9)

[warning] Unused function argument: `email_to`

(ARG001)